gui/src/components/Taipy/Chart.tsx

```diff
@@ -385,6 +385,13 @@ const Chart = (props: ChartProp) => {
                           ret.z = baseZ;
                       }
                   }
+                  // Hack for treemap charts: create a fallback 'parents' column if needed
+                  // This works ONLY because 'parents' is the third named axis
+                  // (see __CHART_AXIS in gui/utils/chart_config_builder.py)
+                  else if (config.types[idx] === "treemap" && ("values" in ret)) {
+                      ret["parents"] = Array((ret["values"] as Array<number>).length).fill("")
```

Comment:
ret["parents"] => ret.parents
ret["values"] => ret.values (as Array might not be needed if tested)

Reply:
Far better, thx